runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java

```diff
@@ -309,7 +309,7 @@ private boolean isAddressToBackpointerResolved(long startAddress, UUID streamId)
             }
         }
 
-        if (!runtime.getParameters().isBackpointersDisabled() && d.hasBackpointer(streamId)) {
```

Comment:
@annym why do we even care about backpointers here?

Reply:
This is an optimization to avoid going to the sequencer if the remaining space is locally resolved already, but the check on that flag was just to be consistent with the fact that it indicated that backpointers might not exist, given that it is always the case we don't need it.